### PR TITLE
Fixed the highlights checking bug

### PR DESCRIPTION
**Pull request by catalincodes**
- Fix highlight checking bug
- Restructure highlight command for easier debugging

### CODE REVIEW

File: eventActions/highlightActions.js [hunk 1]
```diff
@@ -70,7 +71,7 @@ class highlightActions {
 					}
 				}
 				if(contains){
-					const user = client.users.get(currentId);
+					const user = await client.users.cache.get(currentId);
```
Comment (debashisbiswas) [inline eventActions/highlightActions.js]: Do we need `async/await` for this? Correct me if I'm wrong, but I don't think `get` is an async function.

File: commands/highlight.js [hunk 2]
```diff
@@ -8,115 +8,149 @@ const errHandler = err => {
...
 
 	if(keywords.length === 0){
-		const highlightsHelp = new Discord.MessageEmbed()
-			.setColor('#FFEC09')
-			.setTitle(`${highlightsEmote} Knights of Academia Highlight Help ${highlightsEmote}`)
-			.setDescription('Here are some commands to help you out with highlights!')
-			.addField('Add a highlight', '`!highlight add <word/phrase>`')
-			.addField('Remove a highlight', '`!highlight remove <word/phrase>`')
-			.addField('List your highlights', '`!highlight list`');
+		const highlightsHelp = getHelpReply();
```
Comment (debashisbiswas) [inline commands/highlight.js:139]: Nice job splitting all of these into functions, these make the code much more readable :)

File: eventActions/highlightActions.js (hunk 1 above)
Comment (catalincodes) [reply eventActions/highlightActions.js]: You were correct. The await is redundant as `client.users.cache` is a map and as such is not an async call. The await has been removed everywhere it was used.

**Review by debashisbiswas — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**